site/content/entry/yuzu-progress-report-feb-2022/index.md

```diff
@@ -0,0 +1,285 @@
+If you place the mouse over the dotted hyperlink you get the current status, title, number and merge date, for example, {{< gh-hovercard "7969" "here" >}}.
+
+The different possible statuses are:
+
+{{< imgs
+	"./merge.png| There are a few more, but these 3 are the most important"
+  >}}
+
+Red: Pull request closed, it won’t be part of any build.
+Purple: Merged to master, it already is on Mainline, or will be in 24 hours at most, if our buildbot doesn’t die in the process!
+Green: Pull request open, not yet added to Mainline unless it has the `mainline-merge` tag, check the pull request link to confirm.
+
+You can come back to the articles at any time from now on to see if a feature has been merged to mainline already, as these cards will update dynamically accordingly. 
+Keep in mind that this is not an immediate process. 
+A merged pull request will be added to mainline on the next autobuild, so it may take up to 24 hours to see a change in place.
+
+With that out of the way, let’s get started.
+
+## Graphical fixes, and how to increase performance
+
+Due to problems with the RADV drivers on Linux, using the Vulkan API would crash yuzu when run on AMD's RDNA2 graphics cards.
+Most notably, this problem affected the Steam Deck, as reported by users who tried testing our emulator in their devices.
+
+It was determined that the crash happened because of `VK_EXT_vertex_input_dynamic_state`, a Vulkan extension used to minimize the number of pipeline objects needed during the shader compilation process.
+These structures are massive, and sometimes, a game may want to change only a small portion of the contents held in them, such as the vertex input state.
+Using this extension allows the API to dynamically change these members in the structure with a simple function call, eliminating the need to create a whole new object, thus, reducing the amount of resources used by the operation.
+
+While this bug is being resolved upstream, [epicboy](https://github.com/ameerj) pushed a PR to {{< gh-hovercard "7953" "blacklist the extension" >}} on RDNA2 devices running RADV drivers, so yuzu does not outright crash.
+
+But there was still one more driver with problems for us out there.
+
+Mesa's ANV driver on Linux does not support the texture format `VK_FORMAT_B5G6R5_UNORM_PACK16`, and this caused yuzu to crash when running a game that made use of it, such as `Bowser's Fury`.
+To bypass this problem, [voidanix](https://github.com/voidanix) implemented a solution that emulates `BGR565` textures by {{< gh-hovercard "7913" "swizzling the RGB565 format" >}} instead.
+
+In a similar vein, [Morph](https://github.com/Morph1984) implemented a {{< gh-hovercard "7948" "missing vertex format" >}}, used by the title `パワプロクンポケットR` (Power Pro Kun Pocket R).
+
+[asLody](https://github.com/asLody), also implemented a missing {{< gh-hovercard "7939" "framebuffer format" >}}, used by `Yokai-watch 4`.
+
+{{< imgs
+	"./ykw4.png| The real name technically is 妖怪ウォッチ4 ぼくらは同じ空を見上げている"
```

Comment:
```suggestion
	"./ykw4.png| The original name is technically 妖怪ウォッチ4 ぼくらは同じ空を見上げている"
```

Reply:
If we're keeping the runes, then I personally would prefer to add a transcription, since not everybody can read moonrunes.
`Youkai Watch 4 Bokura wa Onaji Sora wo Miageteiru`
Or perhaps just using the transcription directly would be more suitable? Or maybe, a localization of the title?
`Youkai Watch 4: We all look up to the same sky`, or whatever the developers tried to do whenever they gave the game this purple title. Not a fan of localizing it since it's not official, though.